### ISSUE TITLE
`Sample` no longer accepts `evals::Int`

### ISSUE THREAD

**Issue by gdalle** [LilithHafner/Chairmarks.jl#156]
Hi @LilithHafner,
Just so you know, release 1.3.0 seems to have broken my DifferentiationInterface tests because I used to pass `evals=0` to `Sample` and this does not fly anymore? No big deal of course, just wanted to know if it was on purpose or not.

**Comment by LilithHafner** [comment]
I'm seeing

```
MethodError: no method matching Chairmarks.Sample(::Int64, ::Float64, ::Float64, ::Float64, ::Float64, ::Float64, ::Float64, ::Float64, ::Float64)
```

In the CI logs for `DifferentiationInterface` which indicates that you are constructing `Sample`s with all nine arguments enumerated. I can confirm that I broke that in 1.3.0. I'll fix it in 1.3.1. Thanks for reporting.

**Comment by LilithHafner** [comment]
(You can still use `Sample(evals=0)` in 1.3.0, just not `Sample(0, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN)`)

**Comment by LilithHafner** [comment]
I've attempted to fix this in 1.3.1 and triggered [registration](https://github.com/JuliaRegistries/General/pull/120506). Please let me know if you run into any other issues :)

**Comment by gdalle** [comment]
thanks!
